deploy/helm/sumologic/conf/metrics/fluentd/metrics.conf

```diff
@@ -43,6 +43,7 @@
     cache_ttl  {{ .Values.fluentd.metadata.cacheTtl | quote }}
     cache_refresh {{ .Values.fluentd.metadata.cacheRefresh | quote }}
     cache_refresh_variation {{ .Values.fluentd.metadata.cacheRefreshVariation | quote }}
+    cache_refresh_apiserver_request_delay {{ .Values.fluentd.metadata.cache_refresh_apiserver_request_delay | quote }}
```

Comment:
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/conf/logs/fluentd/logs.enhance.k8s.metadata.filter.conf - add this here as well, please.